build.sbt

```diff
@@ -54,9 +51,9 @@ organization := "edu.berkeley.cs"
 
 version := "1.5-SNAPSHOT"
 
-scalaVersion := "2.12.10"
+scalaVersion := "2.13.4"
```

Comment:
In other repos, we haven't changed the default version yet, ie. add `2.13` to `crossScalaVersion` but keep this as `2.12`. It shouldn't really matter either way though and maybe once 2.13 is in Chisel we should start using it as the default version.

Reply:
Default compile version is 2.12